examples/plugins/job-pre-post/mail/avocado_job_mail.py

```diff
@@ -3,7 +3,7 @@
 from email.mime.text import MIMEText
 
 from avocado.core.settings import settings
-from avocado.plugins.base import JobPre, JobPost
+from avocado.core.plugin_interfaces import JobPre, JobPost
```

Comment:
Welcome fix, could you please improve the commit message? I'll cherry-pick it then.